EIPS/eip-5573.md

```diff
@@ -0,0 +1,273 @@
+{
+   "att":{
+      "https://example.com/pictures/":{
+         "crud/delete": [],
+         "crud/update": [],
+         "other/action": []
+      },
+      "mailto:username@example.com":{
+          "msg/receive": [{
+              "max_count": 5,
+              "templates": ["newsletter", "marketing"]
+          }],
+          "msg/send": [{ "to": "someone@email.com" }, { "to": "joe@email.com" }],
+      }
+   },
+   "prf":["bafybeigk7ly3pog6uupxku3b6bubirr434ib6tfaymvox6gotaaaaaaaaa"]
+}
+```
+
+In the example above, the delegee is authorized to perform the actions `crud/update`, `crud/delete` and `other/action` on resource `https://example.com/pictures` without limitations for any. Additionally the delegee is authorized to perform actions `msg/send` and `msg/recieve` on resource `mailto:username@example.com`, where `msg/send` is limited to sending to `someone@email.com` or `joe@email.com` and `msg/recieve` is limited to a maximum of 5 and templates `newsletter` or `marketing`. Note, the delegee can invoke each action individually and independently from each other in the RS. Additionally the ReCap Capability Object contains some additional information that the RS will need during verification. The responsibility for defining the structure and semantics of this data lies with the RS. These action and restriction semantics are examples not intended to be universally understood.
+
+It is expected that RS implementers define which resources they want to expose through ReCap Details Objects and which actions they want to allow users to invoke on them.
+
+This example is expected to transform into the following `recap-transformed-statement` (for `URI` of `https://example.com`):
+
+```text
+I further authorize did:key:example to perform the following actions on my behalf: (1) "crud": "delete", "update" for "https://example.com/pictures". (2) "other": "action" for "https://example.com/pictures". (3) "msg": "recieve", "send" for "mailto:username@example.com".
+```
+
+This example is also expected to transform into the following `recap-uri`:
+
+```text
+urn:recap:eyJhdHQiOnsiaHR0cHM6Ly9leGFtcGxlLmNvbS9waWN0dXJlcyI6eyJjcnVkL2RlbGV0ZSI6W10sImNydWQvdXBkYXRlIjpbXSwib3RoZXIvYWN0aW9uIjpbXX0sIm1haWx0bzp1c2VybmFtZUBleGFtcGxlLmNvbSI6eyJtc2cvcmVjaWV2ZSI6W3sibWF4X2NvdW50Ijo1LCJ0ZW1wbGF0ZXMiOlsibmV3c2xldHRlciIsIm1hcmtldGluZyJdfV0sIm1zZy9zZW5kIjpbeyJ0byI6ImpvZUBlbWFpbC5jb20ifV19fSwicHJmIjpbImJhZnliZWlnazdseTNwb2c2dXVweGt1M2I2YnViaXJyNDM0aWI2dGZheW12b3g2Z290YWFhYWFhYWFhIl19
+```
+
+#### ReCap Translation Algorithm
+
+After applying the ReCap Translation Algorithm on a given SIWE message that MAY include a pre-defined `statement`, the `recap-transformed-statement` in a ReCap SIWE message MUST conform to the following ABNF:
+```text
+recap-transformed-statement = statement recap-preamble 1*(" " recap-statement-entry ".")
+   ; see EIP-4361 for definition of input-statement
+recap-preamble = "I further authorize " uri " to perform the following actions on my behalf:"
```

Comment:
As above: are we updating the statement to `stated URI` instead of the in-line URI?